db/migrate/20161219162030_add_index_into_app.rb

```diff
@@ -0,0 +1,10 @@
+require 'mongoid'
```

Comment:
I've added this migration because i've added a new index named text in App model used here: https://github.com/scicco/errbit/blob/f28a38830be523939e92d1d5b06268a2a6a60760/app/models/app.rb#L57 , so if you have an existing application you must generate the App indexes again to use it.